fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -68,8 +68,7 @@ def command(message)
     end
 
     def command_output(message)
-      actual = (message.split("\r").last || "") # as clearing the line will remove the `>` and the time stamp
-      actual.split("\n").each do |msg|
+      message.lines.map(&:chomp).each do |msg|
```

Comment:
Can we a comment above this saying what this does? 😊 I know I'll forget at some point 😇

Reply:
Of course. 

What does it do exactly?

Reply:
This would be my current understanding:
```
# splits the string into lines and then removes n, r, rn from the end of these strings 
```

Reply:
```suggestion
      # Splits the string into lines and then removes EOL characters
      message.lines.map(&:chomp).each do |msg|
```